### PR TITLE
Updating docs 

### PR DESCRIPTION
**Pull request by BusbyActual**
[ENG](https://github.com/avajs/ava/pull/1727)

### CODE REVIEW

File: fr_FR/docs/recipes/jspm-systemjs.md [hunk 1]
```diff
@@ -19,11 +16,15 @@ Cette recette a été testée uniquement avec JSPM v0.17.0-beta.22, mais ceci de
 
 ### Babel
 
-Configurez votre .babelrc pour travailler avec AVA, si ce n'est pas déjà fait. REMARQUE : Vous pouvez garder une configuration supplémentaire dans vos fichiers de configuration de JSPM pour remplacer ces paramètres pendant le regroupement et la construction.
+Configurez votre .babelrc pour travailler avec AVA, si ce n'est pas déjà fait. REMARQUE : Vous pouvez garder une configuration supplémentaire dans vos fichiers de configuration de JSPM pour remplacer ces paramètres pendant le regroupement et la construction. Vous aurez besoin d'installer le [@babel/preset-env](https://www.npmjs.com/package/@babel/preset-env) comme une dépendance de dev.
```
Comment (forresst) [inline fr_FR/docs/recipes/jspm-systemjs.md]: **Replace:** 
Vous aurez besoin d'installer le [@babel/preset-env](https://www.npmjs.com/package/@babel/preset-env) comme une dépendance de dev.
**By :**
À partir de babel 7, [@babel/ preset-env](https://www.npmjs.com/package/@babel/preset-env) transformera le code comme babel-preset-esXXXX le faisait précédemment.

**Review by forresst — COMMENTED** [review]
Thanks

**Comment by BusbyActual** [comment]
Fixed grammar and rebased.

**Comment by forresst** [comment]
Perfect. I will do the merge when it will be done on the repository of AVA in English

**Comment by BusbyActual** [comment]
https://github.com/avajs/ava/pull/1727 is closed. I patched last changes and rebased.

**Review by BusbyActual — COMMENTED** [review]
confirmed latest commit on file as 29d2c5d3df3895859c7f4d531e57c58ecdd6cba3

**Comment by forresst** [comment]
Thanks @BusbyActual 
I made some adaptations to be identical to the original version

**Pull request merged**